tuxemon/event/actions/npc_wander.py

```diff
@@ -66,8 +89,12 @@ def move(world: WorldState, npc: NPC) -> None:
             exits = world.get_exits(origin)
             if exits:
                 if origin != coords:
-                    npc.path = [random.choice(exits)]
-                    npc.next_waypoint()
+                    path = random.choice(exits)
+                    if not output or path in output:
+                        npc.path = [path]
+                        npc.next_waypoint()
+                    else:
```

Comment:
You should also be able to totally remove this else statement, and even the next one: it's the end of the method, and there is nothing to return.
Let's the method end by itself, no need for extra lines doing nothing.

Reply:
ok